cas-server-support-ldap/src/main/java/org/apereo/cas/authorization/generator/LdapAuthorizationGenerator.java

```diff
@@ -25,12 +24,12 @@
 /**
  * Provides a simple {@link AuthorizationGenerator} implementation that obtains user roles from an LDAP search.
  * Two searches are performed by this component for every user details lookup:
- *
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [<p> tag should be placed immediately before the first word, with no space after.](https://www.codacy.com/app/mmoayyed/cas/file/2619866026/issues/source?bid=3256115&fileBranchId=3313891#l27)